setup.py

```diff
@@ -0,0 +1,59 @@
+import pathlib
+
+from setuptools import find_packages, setup
+
+import versioneer
+
+LONG_DESCRIPTION = """
+Ibis_dask is a dask plugin for Ibis.
+
+Ibis is a productivity-centric Python big data framework.
+See http://docs.ibis-project.org for more information.
```

Comment:
```suggestion
See http://ibis-project.org for more information.
```

If you copied this from Ibis, do you mind opening a PR to update it in Ibis too? This url changed some time ago.